app/gui/src/project-view/components/DockPanel.vue

```diff
@@ -67,11 +68,11 @@ const tabStyle = {
               class="tab"
               :style="tabStyle"
             >
-              <ToggleIcon
+              <SvgButton
                 :modelValue="currentTab == tab"
```

Comment:
`SvgButton` doesn't take a `modelValue`.

Reply:
Better now?

Reply:
Yes, looks good! `modelValue` was working even though it shouldn't have been because Vue passed the unrecognized attribute to the root of the component, and `SvgButton` is implemented based on the same component as `ToggleIcon`. The new solution with `activeTab` works much more straightforwardly.